.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
```

Comment:
I don't think there's a need to check for both node versions, we are only statically checking the code.